api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -187,10 +186,8 @@
 
   @Override
   public ResponseEntity<CriteriaListResponse> getCriteriaById(Long cdrVersionId, Long id) {
-
-    //TODO: really implement this
     cdrVersionService.setCdrVersion(cdrVersionDao.findOne(cdrVersionId));
-    final List<Criteria> criteriaList = criteriaDao.findCriteriaByType("demo");
+    final List<Criteria> criteriaList = criteriaDao.findCriteriaById(id);
```

Comment:
would this be for a parent?

Reply:
This works for parents as well as children.

Reply:
ok